dataset/firebase-export-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "9.16.0",
+  "firestore": {
+    "version": "1.13.1",
+    "path": "firestore_export",
+    "metadata_file": "firestore_export/firestore_export.overall_export_metadata"
+  }
+}
```

Comment:
Missing extra new-line 👀

Reply:
Yes! It's the auto-generated Firestore dataset by the emulator I didn't check it. Oopsy